dex/networks/erc20/contracts/ERC20SwapV0.sol

```diff
@@ -0,0 +1,209 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+// pragma should be as specific as possible to allow easier validation.
+pragma solidity = 0.8.6;
+
+interface ERC20 {
+    function transfer(address recipient, uint256 amount) external returns (bool);
+    function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+}
+
+
+contract ERC20Swap {
+    bytes4 private constant TRANSFER_FROM_SELECTOR = bytes4(keccak256("transferFrom(address,address,uint256)"));
+	bytes4 private constant TRANSFER_SELECTOR = bytes4(keccak256("transfer(address,uint256)"));
```

Comment:
I think these can be removed.  Leftovers from that stx template?

Reply:
Yes this needs to be removed.